lib/manifest.go

```diff
@@ -116,3 +116,17 @@ func ParseManifests(r io.Reader) ([]Manifest, error) {
 		manifests = append(manifests, m)
 	}
 }
+
+// Metav1CreateOptions returns default create options for new objects
+func Metav1CreateOptions() metav1.CreateOptions {
+	return metav1.CreateOptions{
+		FieldManager: "cluster-version-operator",
```

Comment:
this default to the user-agent. Is the user-agent wrong in CVO ?

Reply:
Ah, right, its being set, sorry